inspirehep/modules/search/static/js/search/app.js

```diff
@@ -22,9 +22,6 @@
  */
 
 require([
-    'angular',
```

Comment:
I can't see anything strange happening when I remove them...Do you think that they should be there so as to be initialized when the app is created ?